simple_network_sim/network_of_populations/__init__.py

```diff
@@ -286,7 +292,10 @@ def distributeContactsOverAges(
         totalSus += sus
 
     if totalSus < newContacts:
-        logger.error("totalSus < incoming contacts (%s < %s) - adjusting to totalSus", totalSus, newContacts)
+        desc = f"totalSus < incoming contacts ({totalSus} < {newContacts}) - adjusting to totalSus"
+        logger.error(desc)
+        if issues is not None:
+            issues.append(standard_api.Issue(description=desc, severity=IssueSeverity.HIGH.value))
```

Comment:
Because `issues` is an optional parameter. If it's not available, then the issues are not recorded. I did this to limit the number of changes, but if you think it's better to have it as a required parameter I can change. In general, I prefer when all parameters are required :)